.github/workflows/ios-build.yml

```diff
@@ -12,10 +12,10 @@ permissions:
 
 jobs:
   build-ios:
-    runs-on: macos-15-xlarge
+    runs-on: macos-15
```

Comment:
FYI because the 2.0.2 was squash-merged into master, and then 2.0.3 was branched off of 2.0.2, some of this diff is incorrect because the changes are actually already in master. Including this one.